gme/Spc_Emu.cpp

```diff
@@ -8,27 +8,27 @@
 #include <algorithm>
 
 #ifdef RARDLL
-#ifndef _WIN32
-#define PASCAL
-#define CALLBACK
-#define UINT unsigned int
-#define LONG long
-#define HANDLE void *
-#define LPARAM uintptr_t
-#else
-#ifndef WIN32_LEAN_AND_MEAN
-#define WIN32_LEAN_AND_MEAN
-#endif
-#include <windows.h>
-#endif
-#if defined RAR_HDR_UNRAR_H
-#include <unrar.h>
-#elif defined RAR_HDR_DLL_HPP
-#include <dll.hpp>
-#endif
-#ifndef ERAR_SUCCESS
-#define ERAR_SUCCESS 0
-#endif
+#  ifndef _WIN32
+#    define PASCAL
+#    define CALLBACK
+#    define UINT unsigned int
+#    define LONG long
+#    define HANDLE void *
+#    define LPARAM uintptr_t
+#  else
+#    ifndef WIN32_LEAN_AND_MEAN
+#      define WIN32_LEAN_AND_MEAN
+#    endif
+#    include <windows.h>
+#  endif
+#  if defined RAR_HDR_UNRAR_H
+#    include <unrar.h>
+#  elif defined RAR_HDR_DLL_HPP
+#    include <dll.hpp>
+#  endif
+#  ifndef ERAR_SUCCESS
+#    define ERAR_SUCCESS 0
+#  endif
```

Comment:
Ew ...

Reply:
Can you elaborate?

Reply:
this doesn't really help with anything other than adding unnecessary crowd to the patchset

Reply:
I don't mind about code style tweaks that improves visuality, but, suggested to do them independently from functionality related commits.

Reply:
Is it something that you would want to have implemented all throughout the code base?

I went with 2 spaces for the indentation. The more conventional way seems to be just 1 space. Maybe 1 space would be better as a consistent style all throughout.